services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py

```diff
@@ -78,39 +104,41 @@ async def docker_compose_up(
 
     [SEE docker-compose](https://docs.docker.com/engine/reference/commandline/compose_up/)
     """
-
+    # building is a security risk hence is disabled via "--no-build" parameter
     result = await _write_file_and_spawn_process(
         compose_spec_yaml,
         command=f'docker-compose --project-name {settings.DYNAMIC_SIDECAR_COMPOSE_NAMESPACE} --file "{{file_path}}" up'
         " --no-build --detach",
-        process_termination_timeout=timeout,
+        process_termination_timeout=None,
     )
     return result  # type: ignore
 
 
 async def docker_compose_restart(
-    compose_spec_yaml: str, settings: ApplicationSettings, timeout: int
+    compose_spec_yaml: str, settings: ApplicationSettings
 ) -> CommandResult:
     """
     Restarts running containers (w/ a timeout)
 
     [SEE docker-compose](https://docs.docker.com/engine/reference/commandline/compose_restart/)
     """
-    assert timeout, "timeout here is mandatory"
+    default_compose_restart_timeout = 10
```

Comment:
10s for stopping and restarting containers?? that looks a bit short.

Reply:
That's the default from docker-compose. They do not care about the state of your containers they send a sigterm and wait for 10seconds, afterwards a  sigkill comes.